src/test_l1_dsVideoPort.c

```diff
@@ -3200,7 +3200,7 @@ void test_l1_dsVideoPort_positive_dsGetColorDepth(void) {
         status = dsGetColorDepth(handle, &colorDepth1);
         UT_ASSERT_EQUAL(status, dsERR_NONE);
         // Step 04: Compare the value with values from profile file
-        UT_ASSERT_EQUAL(colorDepth1, gDSvideoPort_color_depth);
+        UT_ASSERT_EQUAL(colorDepth1, getColorDepth());
```

Comment:
rather this we need update the profile with color_depth number 
if you see the https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/main/profiles/source/Source_4K_VideoPort.yaml#L13

Reply:
Can you consider this to update the right value in profile and suggest the same with MTK guys on jira.

Reply:
done

Reply:
i didn't see any change in profile file

Reply:
https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/9723390ddffea922ec08c399cf0998b3cccc4db1/profiles/sink/Sink_4K_VideoPort.yaml#L13

Updated value in sink to 8  (dsDISPLAY_COLORDEPTH_AUTO)